specs/jsonschema-core.md

```diff
@@ -569,11 +601,19 @@ the keyword's value. Alternatively, an applicator may refer to a schema
 elsewhere in the same schema document, or in a different one. The mechanism for
 identifying such referenced schemas is defined by the keyword.
 
-Applicator keywords also define how subschema or referenced schema boolean
-[assertion](#assertions) results are modified and/or combined to produce the
-boolean result of the applicator. Applicators may apply any boolean logic
-operation to the assertion results of subschemas, but MUST NOT introduce new
-assertion conditions of their own.
+Applicator keywords also behave as assertions, using the assertion results of
+each subschema or referenced schema of the keyword. These boolean results are
+modified (e.g. the `not` keyword negates its subschema's assertion) and/or
+combined (e.g. `allOf` takes the conjunction of its subschemas' assertions)
+to produce the boolean result of the applicator.
+any boolean logic operation to the assertion results of subschemas, but SHOULD
+NOT introduce new assertion conditions of their own.[^2]
```

Comment:
Looks like something broke in the last edit. There's a floating sentence fragment at the end.

(Otherwise I like the new wording.)

Reply:
Ah, yeah, that was pulling in a suggestion from @notEthan.  The start of the next sentence was omitted.